src/NServiceBus.Persistence.DynamoDB/Config/DynamoDBPersistenceConfig.cs

```diff
@@ -9,6 +9,11 @@
     /// </summary>
     public static class DynamoDBPersistenceConfig
     {
+        //TODO any reason to not use a static table across all endpoints?
+        internal const string SharedTableName = "NServiceBus.Storage";
```

Comment:
thinking more about this: our outbox records need to be capable of handling an event with the same id on multiple endpoints sharing the same table! let's try to write a test.

Reply:
made a dedicated action item on the board for that.